### DISCUSSION TITLE
What configurations does Konsave saves? From which parts?

### DISCUSSION THREAD

**Discussion by AliAlmasi** [Prayag2/konsave#114] [Q&A]
I'd like to know what configurations does Konsave actually saves?

**Comment by geniusbat** [comment]
Hello!

By default konsave exports all the following files for each path (this is also assuming your desktop environment is KDE, in any other case konsave doesn't provide any default configuration):
**For path "~/.config", will export all folders/files inside named:** 
- gtk-2.0
- gtk-3.0
- gtk-4.0
- kate
- Kvantum
- latte
- dolphinrc
- katerc
- konsolerc
- kcminputrc
- kdeglobals
- kglobalshortcutsrc
- klipperrc
- krunnerrc
- kscreenlockerrc
- ksmserverrc
- kwinrc
- kwinrulesrc
- plasma-org.kde.plasma.desktop-appletsrc
- plasmarc
- plasmashellrc
- gtkrc
- gtkrc-2.0
- lattedockrc
- breezerc
- oxygenrc
- lightlyrc
- ksplashrc
- khotkeysrc
- systemsettingsrc
- kded5rc
- kconf_updaterc
- kscreenrc
- plasma-systemmonitorrc
- spectaclerc
- plasmanotifyrc
- plasma-localerc
- plasma_calendar_holiday_regions
- xdg-desktop-portal-kderc
- kwinoutputconfig.json

**For path "~/.local/share/kxmlgui5", will export all folders/files inside named:** 
- systemsettings
- spectacle
- plasma-systemmonitor
- dolphin
- konsole
- yakuakerc

**For path "~/.local/share", will export all folders/files inside named:** 
- plasma
- kwin
- konsole
- fonts
- color-schemes
- aurorae
- cons
- wallpapers
- yakuake

**For path "~/", will export all folders/files inside named:** 
- .fonts
- .themes
- .icons

I hope this message helps!
